sulley/sessions.py

```diff
@@ -551,8 +551,13 @@ def error_handler (e, msg, target, sock=None):
             # if fuzzing is not finished, web interface thread will catch it
             if self.total_mutant_index == self.total_num_mutations:
                 import signal
-                while True:
-                    signal.pause()
+                try:
+                    while True:
+                        signal.pause()
+                except AttributeError:
+                    # signal.pause() is missing for Windows; wait 1ms and loop instead
+                    while True:
+                        time.sleep(1)
```

Comment:
Yeah I think so too. Honestly looping and sleeping probably isn't the right solution here, but I don't have the cycles to think of something better :-/